docs/release/condor-itb-testing.md

```diff
@@ -72,11 +72,15 @@ directory to get the test directories.
 
 1. Change into the `1-direct-jobs` subdirectory
 
-2. If there are old result files in the directory, remove them:
+1. If there are old result files in the directory, remove them:
 
         :::console
         make distclean
 
+    **Note:** Use the `clean` target to remove intermediate result and log files generated by a workflow run but
```

Comment:
We should use `!!! note` here

Reply:
Not sure it deserves that much highlighting…